atomicfu-transformer/src/main/kotlin/kotlinx/atomicfu/transformer/AtomicFUTransformer.kt

```diff
@@ -718,45 +732,70 @@ class AtomicFUTransformer(
                     return iv
                 }
             }
+            if (f.isArray && iv.name == GET_SIZE) {
+                if (!vh) {
+                    // map to j.u.c.a.Atomic*Array length()
+                    iv.owner = descToName(f.fuType.descriptor)
+                    iv.name = "length"
+                } else {
+                    // map to MethodHandles.arrayLength
+                    val getStaticMH = FieldInsnNode(
```

Comment:
We can just use `arraylength` bytecode here. There's no need to use a MethodHandle for it.

Reply:
Got it, no need to use MethodHandle for this. I've erased mapping to MethodHandles.arrayLength and generating the static field.